sdk/eventhub/azure-eventhub/azure/eventhub/_consumer.py

```diff
@@ -172,6 +173,7 @@ def _create_handler(self, auth: Union[uamqp_JWTTokenAuth, JWTTokenAuth]) -> None
             desired_capabilities=desired_capabilities,
             streaming_receive=True,
             message_received_callback=self._message_received,
+            use_tls=self._use_tls,
```

Comment:
sorry should've caught this earlier, but since this has already been set in config, could we use config everywhere at the sdk level?

and I think you could also just remove passing this in here and the self._use_tls =... altogether, since config is being passed to create_receive_client anyway + since the change is only being added to pyamqp, we could pass it in directly there.

`return ReceiveClient(
...,
use_tls=config.use_tls,
**kwargs
`